### ISSUE TITLE
Can not discover ProjectConfig in Java9

### ISSUE THREAD

**Issue by kintomiko** [kotlintest/kotlintest#258]
somehow these piece of code can not be discovered in Java9, the same works fine in Java8
`11:26:54.257 [main] WARN  org.reflections.Reflections - given scan urls are empty. set urls in the configuration`

```
object GlobalConfiguration : ProjectExtension {

    private var db: DB? = null

    override fun beforeAll() {
        ...
    }

    override fun afterAll() {
        ...
    }

}

object Config : ProjectConfig() {
    override val extensions = listOf(GlobalConfiguration)
}
```

**Comment by sksamuel** [comment]
This is fixed in 3.0.0 RC1.
If you have the time, please give it a try. There are some breaking changes that should be aware of by reading the changelog here: https://github.com/kotlintest/kotlintest/blob/master/CHANGELOG.md

**Comment by kintomiko** [comment]
@sksamuel  all good! Thanks for the fix
